### PR TITLE
COMMERCE-12118

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://liferay.atlassian.net/browse/COMMERCE-12118

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Comment by lorenzolapescara** [comment]
@Brunno0liveira both tests are not covering the bug scenarios, please update the tests to execute the correct steps based on the ticket's description. Thanks!

cc @victoroscuro

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase [hunk 1]
```diff
@@ -1817,6 +1817,250 @@ definition {
...
+
+			CommerceNavigator.gotoTableListEntry(entryName = "ABS Sensor");
+
+			CommerceEntry.gotoMenuTab(menuTab = "Options");
+
+			CommerceRelations.assignSingleRelationToProductsEntry(
+				entryName = "Colour",
+				title = "Add Option");
+
+			Refresh();
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase:1876]: Please avoid using the Refresh function, thanks!

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase [hunk 2]
```diff
@@ -1817,6 +1817,250 @@ definition {
...
+
+			CommerceProductImages.addProductImages(
+				imageTitles = "Black,White",
+				optionValues = "Black,White",
+				validationImage = "true");
+
+			Alert.viewSuccessMessage();
+		}
+
+		task ("When buyer user logs in navigating in Minium site") {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase:1896]: When the buyer user logs in Minium site

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase [hunk 3]
```diff
@@ -1817,6 +1817,250 @@ definition {
...
+
+		task ("Then the buyer place the order with the product") {
+			CommerceAcceleratorsInitializer.gotoMiniumProductDetailsPage(productName = "ABS Sensor");
+
+			Select(
+				key_optionName = "Colour",
+				locator1 = "CommerceFrontStore#FS_PRODUCT_OPTIONS_SELECT",
+				value1 = "Black");
+
+			CommerceProductImages.getProductMediaSrc(imagePath = "//div[contains(@class,'card main-image')]//img[contains(@alt,'Black')]");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase:1913]: This macro is just getting the src field of the image, please add an actual assertion for the image, thanks!

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:commerce-functional

**Review by lorenzolapescara — APPROVED** [review]

**Comment by Brunno0liveira** [comment]
ci:forward

**Pull request closed without merging**